src/core_plugins/kibana/public/management/sections/indices/source_filters/source_filters_controls.html

```diff
@@ -0,0 +1,24 @@
+<div class="controls">
```

Comment:
This file doesn't appear to be used anywhere, accidental duplicate of controls.html?